endpoints/calendar/index.js

```diff
@@ -1,11 +1,57 @@
 import app from '../../server'
 import fridagar from 'fridagar'
 import makeDebug from 'debug'
-import moment from 'moment'
-import { range } from 'lodash'
+import { range, isString } from 'lodash'
 
 const debug = makeDebug('endpoint:calendar')
 
+const canBeInt = intLike => {
+  const num = Number.parseInt(intLike, 10)
+  return !Number.isNaN(num)
+}
+
+export const normalizeParams = (year, month, day) => {
+  // If string parsing failed, reject the promise
+  if (isString(year) && !canBeInt(year)) return { error: 'Year must be a number' }
```

Comment:
Ah I understand now. I'm really for sharing logic with the too since we are not sure if we're 100% invested in GraphQL as of yet.